### ISSUE TITLE
About the remote shell

### ISSUE THREAD

**Issue by Luke9966** [Thibault-69/RAT-Hodin-v2.9#4]
Hi! When I use the remote shell, I find that the cd command doesn't work while other commands work well. Could you help me? And can I open the server files instead of the local files in client computer by just changing the open file function?

**Comment by Thibault-69** [comment]
Hello, you right the cd command dont work as it should, so you must for exemple do like this.
If you want to create a file on the remote desktop don't do cd /home/username/desktop and then touch filename, do instead : 

touch /home/username/desktop/filename

You can for this particular exemple do like this also, this command will create the hacked.txt file and write in it your text :

echo YourText > /home/username/Desktop/hacked.txt

For your second question , i think you can open server file via client open file function already, but maybe you mean to do that remotly.Hmm iam not sure to totaly get your request, anyway if you want to change the open file function feel free to do it.But please if you want to upload Hodin with your function somewhere ask me before, i prefer to add you change to my code and specify your name as the writer of the function, you see ? i worked very hard on Hodin and this program is precious at my eyes i dont want to broke him by adding crazy content.

**Comment by Luke9966** [comment]
Thank you very much! Please be assured that I will never upload files on any platform.

**Comment by Thibault-69** [comment]
Yes you right, with open file option you can only open locals files.

**Comment by Luke9966** [comment]
OK! Thanks!
